backend/src/worker.ts

```diff
@@ -2,3 +2,4 @@ import { nodeMigrator } from "./services/nodeMigrator/index.js";
 import { onchainPublisher } from "./services/onchainPublisher/index.js";
 
 nodeMigrator.start();
+onchainPublisher.start();
```

Comment:
The name of the PR is "remove node publisher" but this is adding a publisher. What am I missing?

Reply:
A little confusing, it fixes the unintended removal of node publisher